src/Microsoft.DotNet.ProjectModel/ProjectContextBuilder.cs

```diff
@@ -37,6 +37,8 @@ public class ProjectContextBuilder
 
         private Func<string, LockFile> LockFileResolver { get; set; }
 
+        private ProjectReader.Settings Settings { get; set; }
```

Comment:
I actually have trouble finding a right name for this.. Any suggestions? ProjectReaderSettings? ProjectContextBuilderSettings?